update: release notes for v0.41.4 (#8748)
cosmos/cosmos-sdk: RELEASE_NOTES.md

# Cosmos SDK v0.41.4 "Stargate" Release Notes

This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.

See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.

## multisign-batch command

Multisign-batch command was added and it allows generating multiple multisig transactions by merging batches of signatures.

## Query tx with multisig addresses

Now the rest endpoint allows to query transactions with multisig addresses.

## Improvements

Major performance improvements in store and balance which will speed up genesis verification and initialization.

> Tendermint was upgraded to v0.34.8.

Reviewer: ```suggestion
Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter.
```

# Cosmos SDK v0.41.4 "Stargate" Release Notes

This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.

See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.

> 

Reviewer 2: quieter Tendermint logs deserves its own subsection, it's imo the most important change in this release

Reviewer: You ok with this? https://github.com/cosmos/cosmos-sdk/pull/8748/files#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50R19

## multisign-batch command

> Multisign-batch command was added and it allows generating multiple musltisig transactions by merging batches of signatures.

Reviewer 2: ```suggestion
Multisign-batch command was added and it allows generating multiple multisig transactions by merging batches of signatures.
```

## Query tx with multisig addresses

Now the rest endpoint allows to query transactions with multisig addresses.

## Improvements

Major performance improvements in store and balance which will speed up genesis verification and initialization.

> Tendermint was upgraded to v0.34.8.

Reviewer 2: ```suggestion
Tendermint was upgraded to v0.34.8, which introduces changes that should mean the logs are much, much quieter. 🎉 
```

Reviewer 2: (or similar wording)

> Genesis now allows 0 coin account balances.

Reviewer: This must be documented more extensively

Author: The genesis account balances?

Reviewer: Yeah

## Bugfixes

> Minor bugfixes were included regarding missing errors and fields on some responses.

Reviewer 2: Also the `keys migrate` command

Reviewer: Added above

# Cosmos SDK v0.41.4 "Stargate" Release Notes

This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.

See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.

## multisign-batch command

Multisign-batch command was added and it allows generating multiple multisig transactions by merging batches of signatures.

## Query tx with multisig addresses

Now the rest endpoint allows to query transactions with multisig addresses.

## Improvements

Major performance improvements in store and balance which will speed up genesis verification and initialization.

> Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter.
> 
> Genesis now allows 0 coin account balances. This means that genesis initialization will not fail if an address with no balance will be included. 

Reviewer: ```suggestion
Genesis now allows 0 coin account balances. This means that genesis initialization will not fail if an address with no balance will be included.

# Tendermint new release

Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter. See [Tendermint's changelog](https://github.com/tendermint/tendermint/blob/v0.34.8/CHANGELOG.md#v0.34.8) for more information.

```

# Cosmos SDK v0.41.4 "Stargate" Release Notes

This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.

See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.

## multisign-batch command

Multisign-batch command was added and it allows generating multiple multisig transactions by merging batches of signatures.

## Query tx with multisig addresses

Now the rest endpoint allows to query transactions with multisig addresses.

## Improvements

Major performance improvements in store and balance which will speed up genesis verification and initialization.

Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter.

Genesis now allows 0 coin account balances. This means that genesis initialization will not fail if an address with no balance will be included. 

## Bugfixes

> Keys migrations command is now functional for offline, multisign and ledger keys.

Reviewer: ```suggestion
The keys migration command (i.e. `keys migrate`) is now functional for offline, multisign, and ledger keys.
```

# Cosmos SDK v0.41.4 "Stargate" Release Notes

This release includes the addition of the multisign-batch command, minor bug fixes, and performance improvements.

See the [Cosmos SDK v0.41.4 milestone](https://github.com/cosmos/cosmos-sdk/milestone/40?closed=1) on our issue tracker for details.

## multisign-batch command

Multisign-batch command was added and it allows generating multiple multisig transactions by merging batches of signatures.

## Query tx with multisig addresses

Now the rest endpoint allows to query transactions with multisig addresses.

## Improvements

Major performance improvements in store and balance which will speed up genesis verification and initialization.

Genesis now allows 0 coin account balances. This means that genesis initialization will not fail if an address with no balance will be included.

> # Tendermint new release
> 
> Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter. See [Tendermint's changelog](https://github.com/tendermint/tendermint/blob/v0.34.8/CHANGELOG.md#v0.34.8) for more information.
> 
> 
> ## Bugfixes
> 
> The keys migration command (i.e. `keys migrate`) is now functional for offline, multisign, and ledger keys.
> 
> Minor bugfixes were included regarding missing errors and fields on some responses.

Reviewer: ```suggestion
## Bugfixes

The keys migration command (i.e. `keys migrate`) is now functional for offline, multisign, and ledger keys.

Minor bugfixes were included regarding missing errors and fields on some responses.

## Tendermint new release

Tendermint was upgraded to v0.34.8. This release of Tendermint introduces various changes that should make the logs much, much quieter. See [Tendermint's changelog](https://github.com/tendermint/tendermint/blob/v0.34.8/CHANGELOG.md#v0.34.8) for more information.
```
